main/tests/cypress/cypress/fixtures/param.tsv

```diff
@@ -0,0 +1,2 @@
+Some parameter	Other parameter	Last parameter
+CONST	123456	12.45
```

Comment:
Can we have a better name for this file ? something more meaningful than `param.tsv` ?

Reply:
done!